syntax = "proto2";

package my_package;

message WheelData {
  required float command_velocity = 1;
  required float status_velocity = 2;
  required float status_position = 3;
}

message Whells {
  repeated WheelData items = 1;
}